### ISSUE TITLE
[North Star Applicant UI] Add styles to review/summary page

### ISSUE THREAD

**Issue by greg-hyde** [civiform/civiform#7687]
Add styles to the review page when specs are available

**Comment by ssandbekkhaug** [comment]
Blocked on [#7935](https://github.com/civiform/civiform/issues/7935) but Sijia mentioned the mocks should be ready today

**Comment by ssandbekkhaug** [comment]
Duplicate of #8794
